Dockerfile

```diff
@@ -1,28 +1,15 @@
-FROM keydonix/parity-instantseal
+FROM keydonix/parity-instantseal-node8
 # TODO: use digest
 
-WORKDIR /build
-
 # TODO: vendor
 RUN apt-get update && apt-get -y install software-properties-common git make && \
 	add-apt-repository ppa:ethereum/ethereum && \
 	apt-get update && \
 	apt-get install -y solc
 
-RUN git clone https://github.com/makerdao/sai.git && \
-	( cd sai && \
-	git checkout 56eed66bb42a485ff0819f4ca227f615b1eb5320 && \
-	git submodule update --init --recursive )
-
-RUN mkdir artifacts && mkdir tools && cd tools && \
-	git clone https://github.com/dapphub/dapp && \
-	(cd dapp && git checkout a426596705be4dfcdd60e7965163453574459dcf) && \
-	make install -C dapp prefix=/build/artifacts
-
-ENV PATH="/build/artifacts/bin:${PATH}"
-
-WORKDIR /build/sai
+COPY . /build/maker-docker-poa
```

Comment:
Consider either adding a `.dockerignore` file or constraining what gets copied to the files you need (e.g., just the source folder).  I think in this case, we want to copy most things, and the default `.dockerignore` includes `.git` and `node_modules` folders, but I am always made uncomfortable by implicit rules like that.

Note: This is incredibly minor and I care incredibly little, and if you don't like it I'll roll over really easily.  😁

Reply:
Nah, i like it, makes the docker build clean, not depending on your own node_modules